source/sessions/driver-sessions.rst

```diff
@@ -1289,10 +1323,23 @@ is needed and, known it will be used, after connection checkout succeeds.
 It is still possible that via explicit sessions or cursors, which hold on to the session they started with, a driver could over allocate sessions.
 But those scenarios are extenuating and outside the scope of solving in this spec.
 
+Appendix
+========
+
+Drivers that DO NOT implement ServerSession pooling
```

Comment:
I believe every driver already pools sessions. Can we remove this section?

Reply:
Is it not worth saving for historical reasons / new drivers? (despite it being a MUST now) 
I'm just thinking it is an important detail that you have to send the command over upon release as opposed to upon close.

Reply:
Maybe add a `for legacy reasons` to the first sentence? I don't have very strong feelings about this.

Reply:
I don't think this section is worth keeping. I don't think there was ever a driver that didn't pool.

Reply:
SGTM, removed.